apps/mobile/src/components/Feed/createVirtualizedFeedEventItems.ts

```diff
@@ -231,13 +231,15 @@ export function createVirtualizedFeedEventItems({
   }
 
   const setVirtualizeItemsForPost = (post: createVirtualizedFeedEventItemsPostFragment$data) => {
+    const uniqueKey = Math.random().toString();
```

Comment:
Added unique key since there are cases that same post could be shown more than once in feed. Its for virtualize

Reply:
ohh good call! 🧠